app/email/service.go

```diff
@@ -141,10 +141,7 @@ type ServiceInterface interface {
 	SendDeactivateAccountEmail(email string, locale, siteURL string) error
 	SendNotificationMail(to, subject, htmlBody string) error
 	SendMailWithEmbeddedFiles(to, subject, htmlBody string, embeddedFiles map[string]io.Reader) error
-	SendAtUserLimitWarningEmail(email string, locale string, siteURL string) (bool, error)
 	SendLicenseUpForRenewalEmail(email, name, locale, siteURL, renewalLink string, daysToExpiration int) error
-	SendUpgradeEmail(user, email, locale, siteURL, action string) (bool, error)
-	SendOverUserLimitWarningEmail(email string, locale string, siteURL string) (bool, error)
 	SendOverUserLimitThirtyDayWarningEmail(email string, locale string, siteURL string) (bool, error)
```

Comment:
I think some of the build errors are occurring because methods need to be deleted from here so that the actual implementation can match this interface.